src/SDK/Common/Export/TransportFactoryInterface.php

```diff
@@ -29,4 +29,7 @@ public function create(
         ?string $cert = null,
         ?string $key = null,
     ): TransportInterface;
+
+    public function type(): string;
+    public function priority(): int;
```

Comment:
Should we be considering this to be a breaking change? 🤔

Example in [exporter-otlp](https://github.com/opentelemetry-php/exporter-otlp/blob/main/OtlpHttpTransportFactory.php).

Reply:
This is a good point, we should consider trying to do this in a non-breaking way. Perhaps another level of abstraction like a `TransportFactoryProvider` & friends which has these extra methods and returns a factory :thinking: